a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -853,38 +853,77 @@ class TagInterfaceForMember(TagInterfaceBaseTask):
 
     @axapi_client_decorator
     def execute(self, member, vthunder):
-        if not member.subnet_id:
-            LOG.warning("Subnet id argument was not specified during "
-                        "issuance of create command/API call for member %s. "
-                        "Skipping TagInterfaceForMember task", member.id)
-            return
-        try:
-            vlan_id = self.get_vlan_id(member.subnet_id, False)
-            self.tag_interfaces(vthunder, vlan_id)
-            LOG.debug("Successfully tagged interface with VLAN id %s for member %s",
-                      str(vlan_id), member.id)
-        except (acos_errors.ACOSException, req_exceptions.ConnectionError) as e:
-            LOG.exception("Failed to tag interface with VLAN id %s for member %s",
+        # Case when batch_update_member is triggered
+        if isinstance(member, list):
+            subnet_list = []
+            for m in member:
+                if m.subnet_id not in subnet_list:
+                    if not m.subnet_id:
```

Comment:
Is it possible to move line 861~875 to a function, and reuse it in else case?
And I think if we move this block to a function, then we can use return to replace the continue here in line 865.

Reply:
Refactored this in https://github.com/a10networks/a10-octavia/pull/524